ReactWindows/ReactNative/Modules/DeviceInfo/DeviceInfoModule.cs

```diff
@@ -62,17 +62,23 @@ public override void Initialize()
         /// </summary>
         public void OnSuspend()
         {
-            ApplicationView.GetForCurrentView().VisibleBoundsChanged -= OnVisibleBoundsChanged;
-            DisplayInformation.GetForCurrentView().OrientationChanged -= OnOrientationChanged;
+            if (!DisplayMetrics.IsBackground)
+            {
+                ApplicationView.GetForCurrentView().VisibleBoundsChanged -= OnVisibleBoundsChanged;
+                DisplayInformation.GetForCurrentView().OrientationChanged -= OnOrientationChanged;
```

Comment:
We need a better solution for this. Perhaps lazy loaded modules can help. Not sure...

Reply:
Lazy loaded modules are useful in case there are native module constants (as is the case here)

---
In reply to: [145262209](https://github.com/Microsoft/react-native-windows/pull/1450#discussion_r145262209) [](ancestors = 145262209)